app/src/main/java/com/javahelp/frontend/activity/ProviderRegistrationActivity.java

```diff
@@ -25,6 +26,10 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.setPdata(providerRegistrationViewmodel);
         binding.setLifecycleOwner(this);
 
+        //setting up spinner to create dropdown item for gender
+        ArrayAdapter<CharSequence> adapter=ArrayAdapter.createFromResource(this, R.array.genders, android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_item);
+        binding.gender.setAdapter(adapter);
```

Comment:
Since it's used in several places, perhaps we could make a `GenderSelector` `View`.